lotusrb.gemspec

```diff
@@ -23,6 +23,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'lotus-router',     '~> 0.4', '>= 0.4.2'
   spec.add_dependency 'lotus-controller', '~> 0.4', '>= 0.4.4'
   spec.add_dependency 'lotus-view',       '~> 0.4', '>= 0.4.3'
+  spec.add_dependency 'lotus-mailer',       '~> 0.1', '>= 0.1.0'
```

Comment:
`'~> 0.1'` is enough :smile_cat: http://stackoverflow.com/questions/3492772/gem-dependencies-versions-meaning